test/integration/authorize.spec.ts

```diff
@@ -0,0 +1,14 @@
+import config from 'config'
+import { PayUClient } from '../../src'
+import { describe, it } from 'mocha'
+import assert from 'assert'
+
+describe('authorization', () => {
+  it('should authorize via real api', async () => {
+    const res = await new PayUClient(config.get('payu')).Authorize()
```

Comment:
are you sure, you want to run test over third party api? In case of unavailable api, you won't be able to merge pull request.

Reply:
yep :) to be sure its ok, its the only way